.github/copilot-instructions.md

```diff
@@ -0,0 +1,116 @@
+# GitHub Copilot Instructions for OpenMS
+
+## Important Build Instructions
+
+**DO NOT attempt to build OpenMS** - The build process is extremely resource-intensive and costly. Building OpenMS requires significant computational resources and time.
```

Comment:
Maybe add "unless asked/prompted to do so"? Otherwise it might refuse even in local agent environments where you have pre built stuff and compilation takes seconds.